### PR TITLE
Team page

### PR DESCRIPTION
**Pull request by wackerow**
## Target / builds on `_status-work`
- #123

### CODE REVIEW

**Comment by wackerow** [comment]
Blocked pending #129

File: app/prover/[teamId]/page.tsx [hunk 1]
```diff
@@ -73,58 +79,85 @@ export default async function ProverPage({ params }: ProverPageProps) {
...
+  const proverTotalFees = completedProofs.reduce(
     (acc, proof) => acc + (proof.proving_cost as number),
     0
   )
-  const avgCostPerProof = proverTotalFees / totalProofs
+  const avgCostPerMgas = proverTotalFees / totalGasProven / 1e6
+
+  const proofs = proofsExtended.map(
+    // eslint-disable-next-line unused-imports/no-unused-vars
+    ({ blocks, ...proof }) => proof as Proof
```
Comment (pettinarip) [inline app/prover/[teamId]/page.tsx:120]: this is not going to end up well, will try to improve the types a bit

File: components/ui/info.tsx
Comment (pettinarip) [inline components/ui/info.tsx:1]: For these type of custom components (exporting granular components), would be nice to add an example as documentation to know the expected structure and how they should be composed.

File: components/Metrics.tsx
Comment (pettinarip) [inline components/Metrics.tsx:1]: 👍🏼

**Review by pettinarip — APPROVED** [review]
Looks good @wackerow. JLYK, I've pushed a few commits to fix conflicts and refactor a bit the proof types.

**Pull request merged**